File: libraryTest/libraryTest/ViewController.m
//
//  ViewController.m
//  libraryTest
//
//  Created by aa64mac on 4/11/15.
//  Copyright (c) 2015 aa64mac. All rights reserved.
//

#import "ViewController.h"
#import "lib2.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    lib2 *l2 = [lib2 alloc];
    int x = [l2 addTest:1 withNumber:2];
    NSLog(@"%d", x);
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOSbeautyOfDevelop/chapter6/6.2.2/dismissViewControllerS/dismissViewControllerS/ViewController.swift
//
//  ViewController.swift
//  dismissViewControllerS
//
//  Created by aa64mac on 8/26/15.
//  Copyright (c) 2015 cc. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func dismissButtonTouchUpInside(sender: AnyObject) {
        self.dismissViewControllerAnimated(true, completion: { () -> Void in
            NSLog("OK")
        })
    }

}

